lib/services/ios-project-service.ts

```diff
@@ -481,5 +558,24 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 	private buildPodfileContent(pluginPodFilePath: string, pluginPodFileContent: string): string {
 		return `# Begin Podfile - ${pluginPodFilePath} ${os.EOL} ${pluginPodFileContent} ${os.EOL} # End Podfile ${os.EOL}`;
 	}
+
+	private generateMobulemap(headersFolderPath: string, libraryName: string): void {
+		let headersFilter = (fileName: string, containingFolderPath: string) => (path.extname(fileName) === ".h" && this.$fs.getFsStats(path.join(containingFolderPath, fileName)).wait().isFile());
+		let headersFolderContents = this.$fs.readDirectory(headersFolderPath).wait();
+		let	headers = _(headersFolderContents).filter(item => headersFilter(item, headersFolderPath)).value();
```

Comment:
I can't think of anything that chould cause such spacing. On my local machine everything looks well formatted. `grunt lint` doesn't yeld any errors.